### PR TITLE
Remove bootstrap from new home page -2

### PR DESCRIPTION
**Pull request by cp-priyanka-g**
Remove Boostrap CSS from below Sections of home page-2

- [x] ClientReview Section
- [x] UserReview section
- [x] Blog Section
- [x] Contribution Section
- [x] CTA section

### CODE REVIEW

File: vue-frontend/src/components/home-new/CTASection.vue [hunk 1]
```diff
@@ -1,68 +1,106 @@
...
+    >
       <img
         :src="backgroundImage"
-        class="render-background"
+        class="tw-w-full tw-h-full tw-top-0 tw-left-0 tw-absolute tw-object-cover tw--z-[1]"
         alt="social-media-background"
       />
-      <div class="container cta-section text-center">
-        <div class="v2-header-3-text canopas-gradient-text">
+      <div
```
Comment (cp-shruti-s) [inline vue-frontend/src/components/home-new/CTASection.vue:11]: add container class here 

Screenshot from 2022-09-12 14-45-51

File: vue-frontend/src/components/home-new/UserReview.vue [hunk 2]
```diff
@@ -1,94 +1,119 @@
 <template>
-  <div class="container overlay">
-    <div class="v2-header-3-text text-center title-width">
+  <div
```
Comment (cp-shruti-s) [inline vue-frontend/src/components/home-new/UserReview.vue:2]: Live website user review section : 
Screenshot from 2022-09-12 14-43-25

and it's your IP address user review section :
Screenshot from 2022-09-12 14-42-45

please fix UI

File: vue-frontend/src/components/home-new/BlogSection.vue [hunk 3]
```diff
@@ -1,59 +1,78 @@
...
+            <div
+              class="md:tw-w-[70%] lg:tw-w-[60%] v2-header-3-text tw-text-center md:tw-text-start"
+            >
               Do we have a team with the right skills?
-              <div class="v2-title-3-text mt-4">
+              <div class="v2-title-3-text tw-mt-6">
                 Well, at least the community says Hell Yeah.
               </div>
-              <div class="v2-title-3-text mt-2">
+              <div class="v2-title-3-text tw-mt-2">
```
Comment (cp-shruti-s) [inline vue-frontend/src/components/home-new/BlogSection.vue:16]: i think bootstrap css class is different to the tailwind css

File: vue-frontend/src/components/home-new/CTASection.vue [hunk 4]
```diff
@@ -108,148 +146,8 @@ export default {
...
-  .cta-section {
-    width: 70%;
-  }
-}
-
-@supports (-webkit-touch-callout: none) {
-  .v2-header-3-text {
-    letter-spacing: -2px;
-  }
+  @apply tw-font-light;
```
Comment (cp-shruti-s) [inline vue-frontend/src/components/home-new/CTASection.vue:151]: This css is not apply as inline css ??

File: vue-frontend/src/components/home-new/UserReview.vue [hunk 5]
```diff
@@ -1,94 +1,119 @@
...
+      >
         <div
           v-for="userReview in usersReviews.slice(0, 3)"
           :key="userReview.id"
-          class="users-review v2-normal-3-text"
+          class="tw-flex-[1_1_100%] md:tw-flex-[1_1_20%] tw-border-[1.5px] tw-border-solid tw-border-[#fa7b78] tw-rounded-[16px] tw-p-5 tw-mt-[15px] tw-mx-[15px] tw-mb-0 tw-flex tw-flex-col tw-justify-center md:tw-text-center md:tw-m-2.5 md:tw-py-5 md:tw-px-[15px] hover:tw-cursor-pointer hover:tw-shadow-[0_1px_6px_2px_rgba(249,129,136,1)] users-review v2-normal-3-text"
+          :class="[userReview.ClassName]"
         >
           {{ userReview.review }}
           <div class="v2-title-3-text mt-3">
```
Comment (cp-shruti-s) [inline vue-frontend/src/components/home-new/UserReview.vue:32]: Apply tailwind css class

File: vue-frontend/tailwind.config.cjs [hunk 6]
```diff
@@ -35,17 +37,30 @@ module.exports = {
...
+        "4xl": "2px 1000px 1px #3d3d3d inset",
       },
+
       lineHeight: {
         11: "2.75rem",
         17: "4.25rem",
         23: "5.75rem",
       },
+
+      fontFamily: {
```
Comment (cp-shruti-s) [inline vue-frontend/tailwind.config.cjs:57]: fontFamily is all ready define ??

Comment (cp-priyanka-g) [reply vue-frontend/tailwind.config.cjs:57]: We have to place it here also, only then the new font will apply.

File: vue-frontend/src/components/home-new/CanopasDescription.vue [hunk 7]
```diff
@@ -36,7 +36,7 @@
         <div class="tw-flex tw-flex-col tw-items-center tw-pt-12">
           <div class="tw-text-center">
             <router-link
-              class="v2-normal-3-text tw-flex tw-items-center tw-py-4 tw-px-[1.1rem] active:tw-scale-[0.98] tw-bg-black-900 tw-text-white hover:tw-bg-transparent hover:tw-text-black-900 tw-rounded-[3rem] tw-text-center tw-border-[1px] tw-border-solid tw-border-[#3d3d3d] tw-bg-white tw-text-black"
+              class="v2-normal-3-text active:tw-scale-[0.98] tw-flex tw-items-center tw-py-4 tw-px-[1.1rem] tw-bg-black-900 tw-text-white hover:tw-bg-transparent hover:tw-text-black-900 tw-rounded-[3rem] tw-text-center tw-border-[1px] tw-border-solid tw-border-[#3d3d3d]"
```
Comment (cp-shruti-s) [inline vue-frontend/src/components/home-new/CanopasDescription.vue:39]: why you are write custom css code over here ?? we have v2-button css

File: vue-frontend/src/components/home-new/ClientReview.vue [hunk 8]
```diff
@@ -74,7 +88,7 @@ export default {
           image: marcus,
           name: "Marcus L.",
           fontColor: "#FFFFFF",
```
Comment (cp-shruti-s) [inline vue-frontend/src/components/home-new/ClientReview.vue:90]: Apply tailwind css text color

File: vue-frontend/src/components/home-new/PortfolioSection.vue [hunk 9]
```diff
@@ -73,10 +73,10 @@ import smilep800w from "@/assets/images/portfolio/smilep-800w.webp";
 import smilep1200w from "@/assets/images/portfolio/smilep-1200w.webp";
 import smilep16000w from "@/assets/images/portfolio/smilep-1600w.webp";
 
-import justly400w from "@/assets/images/portfolio/justly-400w.webp";
-import justly800w from "@/assets/images/portfolio/justly-800w.webp";
-import justly1200w from "@/assets/images/portfolio/justly-1200w.webp";
-import justly16000w from "@/assets/images/portfolio/justly-1600w.webp";
+import nolonely400w from "@/assets/images/portfolio/nolonely-400w.webp";
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/PortfolioSection.vue:76]: Revert these changes

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]
